packages/wallet/ui/src/contexts/Provider.jsx

```diff
@@ -2,6 +2,8 @@
 import { observeIterator } from '@agoric/notifier';
 import { E } from '@endo/far';
 import { useEffect, useState, useReducer } from 'react';
+import { SigningStargateClient } from '@cosmjs/stargate';
```

Comment:
Let's (later) introduce a powerful module that closes over these dependencies and is imported by `Provider.jsx` to keep `Provider.jsx` simpler.

It's fine for now.